src/streams/EditStreamCard.js

```diff
@@ -47,8 +53,12 @@ export default class EditStreamCard extends PureComponent<Props, State> {
   };
 
   render() {
-    const { initialValues, isNewStream } = this.props;
-    const { name } = this.state;
+    const { initialValues, isNewStream, streams } = this.props;
+    const { name, description, isPrivate } = this.state;
+
+    const isInvalidStreamName =
+      caseInsensitiveCompareFunc(name, initialValues.name) !== 0
```

Comment:
Nice that you are reusing a function but this seems a bit convolutrd.
hat would read better as:
```js
name.toLowerCase() === initialValues.name.toLowerCase()
```

That being said, I am not sure this part is even needed.
On a new stream creation, you are comparing with `''`, and on stream edit, you are comparing with stream's previous name which should be valid.

Reply:
Thank you @borisyankov for review.

Indeed, this condition is a little bit counterintuitive, but without it, we would not allow user to change stream name case (ie from "New_Stream" to "NEW_STREAM"). Moreover, the error message would be displayed even if the user did not change stream name field.